src/actions/loadData.js

```diff
@@ -12,42 +12,73 @@ import { hasExtension, getExtension } from "../util/extensions";
 /* TODO: make a default auspice server (not charon) and make charon the nextstrain server. Or vice versa. */
 const serverAddress = hasExtension("serverAddress") ? getExtension("serverAddress") : charonAPIAddress;
 
+/**
+ * Sends a GET response to the `/charon` web API endpoint requesting data.
+ * Throws an `Error` if the response is not successful or is not a redirect.
+ *
+ * Returns a `Promise` containing the `Response` object. JSON data must be
+ * accessed from the `Response` object using the `.json()` method.
+ *
+ * @param {String} prefix: the main dataset information pertaining to the query,
+ *  e.g. 'flu'
+ * @param {Object} additionalQueries: additional information to be parsed as a
+ *  query string such as `type` (`String`) or `narrative` (`Boolean`).
+ */
 const getDatasetFromCharon = (prefix, {type, narrative=false}={}) => {
   let path = `${serverAddress}/${narrative?"getNarrative":"getDataset"}`;
   path += `?prefix=${prefix}`;
   if (type) path += `&type=${type}`;
 
   const p = fetch(path)
     .then((res) => {
-      if (res.status !== 200) {
+      if (!(res.status === 200 || res.status === 301)) {
```

Comment:
In all my testing, the client never actually sees the redirect (301), so should this still be here?